### PR TITLE
resource/aws_synthetics_canary: fix minimum timeout_in_seconds

### PR DESCRIPTION
**Pull request by Kostavro**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes https://github.com/hashicorp/terraform-provider-aws/issues/19289

Output from acceptance testing:

```
-
```

### CODE REVIEW

File: .changelog/19515.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_synthetics_canary: Fix minimum timeout_in_seconds in run_config
```
Comment (ewbankkit) [inline .changelog/19515.txt:2]:
```suggestion
resource/aws_synthetics_canary: Change minimum `timeout_in_seconds` in `run_config` from `60` to `3`
```

**Comment by ewbankkit** [comment]
@Kostavro Thanks for the contribution 🎉 👏.
Everything looks to be in great shape - I just made a minor suggestion on the CHANGELOG entry.

File: .changelog/19515.txt (hunk 1 above)
Comment (Kostavro) [reply .changelog/19515.txt]: Changed to suggested text

**Comment by Kostavro** [comment]
Thanks @ewbankkit . I changed the changelog text as per your suggestion.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

**Pull request merged**